Fix compilation error with the configure option combination
	--with-msw --enable-universal --disable-menus

Signed-off-by: Anders Larsen <al@alarsen.net>
---
 src/msw/taskbar.cpp |    2 ++
 src/msw/window.cpp  |    4 ++++
 2 files changed, 6 insertions(+)

Index: b/src/msw/window.cpp
===================================================================
--- a/src/msw/window.cpp
+++ b/src/msw/window.cpp
@@ -3151,6 +3151,7 @@ WXLRESULT wxWindowMSW::MSWWindowProc(WXU
             break;
 #endif
 
+#if wxUSE_MENUS
         case WM_MENUCHAR:
             // we're only interested in our own menus, not MF_SYSMENU
             if ( HIWORD(wParam) == MF_POPUP )
@@ -3164,6 +3165,7 @@ WXLRESULT wxWindowMSW::MSWWindowProc(WXU
                 }
             }
             break;
+#endif
 
 #ifndef __WXWINCE__
         case WM_POWERBROADCAST:
@@ -5195,6 +5197,7 @@ bool wxWindowMSW::HandleKeyUp(WXWPARAM w
     return GetEventHandler()->ProcessEvent(event);
 }
 
+#if wxUSE_MENUS
 int wxWindowMSW::HandleMenuChar(int WXUNUSED_IN_WINCE(chAccel),
                                 WXLPARAM WXUNUSED_IN_WINCE(lParam))
 {
@@ -5276,6 +5279,7 @@ bool wxWindowMSW::HandleClipboardEvent( 
 
     return GetEventHandler()->ProcessEvent(evt);
 }
+#endif
 
 // ---------------------------------------------------------------------------
 // joystick
Index: b/src/msw/taskbar.cpp
===================================================================
--- a/src/msw/taskbar.cpp
+++ b/src/msw/taskbar.cpp
@@ -171,6 +171,7 @@ bool wxTaskBarIcon::RemoveIcon()
     return Shell_NotifyIcon(NIM_DELETE, &notifyData) != 0;
 }
 
+#if wxUSE_MENUS
 bool wxTaskBarIcon::PopupMenu(wxMenu *menu)
 {
     wxASSERT_MSG( m_win != NULL, _T("taskbar icon not initialized") );
@@ -206,6 +207,7 @@ bool wxTaskBarIcon::PopupMenu(wxMenu *me
 
     return rval;
 }
+#endif
 
 void wxTaskBarIcon::RegisterWindowMessages()
 {

 	  	 
